### DISCUSSION TITLE
Add ability to skin top background on song selection screen

### DISCUSSION THREAD

**Discussion by AsakaJX** [ppy/osu#25606] [Ideas] [answered]
I'm talking about that one with "Group" and "Sort" text's:
![image](https://github.com/ppy/osu/assets/95882188/817b2016-c4ef-4ee5-b609-fcac5fa412ec)
I wanted to make clean song selection screen, but that thing is messing everything up, also it would be better in general for skinning if that forced background is gonna be removed.

**Comment by peppy** [comment, marked as answer]
You are using stable. This repository is for lazer, the next release of osu!.

Skinning the top bar is available. See https://osu.ppy.sh/home/changelog/stable40/20231021.2.
